--- origsrc/a52dec-0.7.4/libao/configure.incl	2002-04-28 05:23:02.000000000 -0500
+++ src/a52dec-0.7.4/libao/configure.incl	2007-02-15 21:38:28.984375000 -0600
@@ -5,7 +5,7 @@
     [  --disable-oss           make a version not using oss])
 if test x"$enable_oss" != x"no"; then
     case "$host" in
-    *-linux*|*-openbsd*|*-freebsd*|*-bsdi*)
+    *-linux*|*-openbsd*|*-freebsd*|*-bsdi*|*-cygwin*)
 	AC_DEFINE([LIBAO_OSS],,[libao OSS support])
 	AC_CHECK_LIB([ossaudio],[_oss_ioctl],
 	    [LIBAO_LIBS="$LIBAO_LIBS -lossaudio"]);;
@@ -38,7 +38,7 @@
     [  --disable-win           make a version not using win audio])
 if test x"$enable_win" != x"no"; then
     case "$host" in
-    *-mingw32*|*-cygwin)
+    *-mingw32*)
 	LIBAO_LIBS="$LIBAO_LIBS -lwinmm"
 	AC_DEFINE([LIBAO_WIN],,[libao win support]);;
     esac
